<launch>

    <param name="use_sim_time" value="true" />

    <!-- DEBUG WITH ROSBAG -->
    <node pkg="rosbag" type="play" name="play" args="/home/matt/rosbag/opt1.bag --delay=1 --clock "/>

    <!-- DECOMPRESSED COMPRESSED TO RAW-->
    <node name="republish" pkg="image_transport" type="republish"  output="screen" args="compressed in:=/raspicam_node/image raw out:=camera/raw_image " />

    <!-- OPTICAL FLOW-->
    <node name="optical_flow" pkg="optical_flow" type="optical_flow_range" output="screen"/>

    <!-- FAKE ULTRASOUND -->
    <node name="fake_us" pkg="optical_flow" type="fake_ultrasound" output="screen"/>

</launch>
